ols/app/endpoints/feedback.py

```diff
@@ -27,7 +119,33 @@ def user_feedback(
     """
     logger.info(
         f"feed back received for conversation '{feedback_request.conversation_id}', "
-        f"feedback blob: {feedback_request.feedback_object}"
+        f"feedback data: {feedback_request.feedback_object}"
+    )
+
+    user_id = auth[0]
+    store_feedback(
+        feedback_request.conversation_id, user_id, feedback_request.feedback_object
     )
 
     return FeedbackResponse(response="feedback received")
+
+
+# TODO: OLS-136 implements the collection mechanism - revisit the need
+# of this endpoint
+@router.delete("")
+def remove_user_feedback(
+    feedback_request: RemoveFeedbackRequest, auth: Any = Depends(auth_dependency)
+) -> FeedbackResponse:
```

Comment:
same comments as the list endpoint.  let's remove this until we need it, and if we need it, the auth permissions need to be different.